### DISCUSSION TITLE
Any roadmap for PyTorch version

### DISCUSSION THREAD

**Discussion by sarthakpati** [DeepRegNet/DeepReg#697] [Ideas]
Thanks for this awesome tool, folks!

I am curious if you would considering a PyTorch version of this tool? Especially considering it is **far** more deployable compared to TF (which is a critical requirement in academia)? Don't want to start a flame-war, but, [this](https://www.reddit.com/r/MachineLearning/comments/m3boyo/d_why_is_tensorflow_so_hated_on_and_pytorch_is/?utm_source=amp&utm_medium=&utm_content=post_body&ampcid=1*re3wx2*cid*YW1wLWx3NTVnTTVfNjI3VlQ5R2w2b21OaVE) 😄.

Cheers,
Sarthak

**Comment by mathpluscode** [comment]
Hi @sarthakpati, thanks for asking! We are working on a PyTorch version in https://github.com/Project-MONAI/MONAI currently. But it gonna take some time and not all features will be available for the first release.

Regarding TensorFlow, one of the objectives of this project is also to make using TensorFlow for registration easier, therefore any comments/ideas/suggestions are welcome!
